site/content/docs/5.2/migration.md

```diff
@@ -71,6 +71,14 @@ If you need more details about the changes, please refer to the [v5.2.3 release]
   +        <a class="stepped-process-link" title="2. Review" aria-current="step">Review</a>
   ```
 
+### Contents
+
+- **Tables**
+  - <span class="badge bg-warning">Warning</span> Every table now has a caption with `.visually-hidden` on it. Please reflect this modification into your websites.
+  - <span class="badge bg-success">New</span> Tables now fully support striped rows and columns.
```

Comment:
I already don't remember what we said about it 😇 but do we display things that are not ODS compatible?

Reply:
Can't remember too >< Can't even remember if we tackled this.

Reply:
We've let small size form controls in the migration note which is not particularly compliant with ODS guidelines so let's let this one too.